### ISSUE TITLE
fatjar cause big jar size

### ISSUE THREAD

**Issue by yoavst** [cbeust/kobalt#56]
```
val p = kotlinProject {
    name = "Kotlin"
    group = "com.yoavst.kotlin"
    artifactId = name
    version = "0.1"
    sourceDirectories = hashSetOf("src/")

    application {
        mainClass = "com.yoavst.kotlin.PackageKt"
    }


    assemble {
        jar {
            fatJar = true
            manifest {
                attributes("main-class", "com.yoavst.kotlin.PackageKt")
            }
        }
    }

}
```

Make a 18MB jar, from 5 small classes. I think it is because it includes `Kotlin-reflect`.

**Comment by cbeust** [comment]
Should be fixed in 0.301:

```
$ ls -l  kobaltBuild/libs/kotlin-simple-0.1.jar
-rw-r--r--  1 beust  1  957422 Nov 27 00:49 kobaltBuild/libs/kotlin-simple-0.1.jar
```

I now include `kotlin-runtime` and `kotlin-stdlib` only, which add up to about 1 meg.

Can you try and report back?

**Comment by cbeust** [comment]
Closing, reopen if it's not fixed.
